Seccion #75, #76, #77 y #78 (Completas) - Proyecto UpTask (Parte 6)

CarlosJCD opened the pull request:
avances en codigo de las secciones #75, #76, #77 y #78 del curso.

CarlosJCD committed:
Video #648 - Seccion #75 (INICIO)

HTML para mostrar los 3 filtros para las tareas

CarlosJCD committed:
Video #649 - Seccion #75

Estilos a la seccion de filtros

CarlosJCD committed:
Video #650 - Seccion #75

Filtrar las tareas segun el input radio seleccionado

CarlosJCD committed:
Video #651 - Seccion #75 (FIN)

Restringiendo la seleccion de filtros segun hayan tareas disponibles con ese filtro

CarlosJCD committed:
Video #652 - Sección #76 (INICIO)

Desplegando y ocultando las barras segun el tamaño de pantalla del dispositivo
Estilos a la barra mobile

CarlosJCD committed:
Video #653 - Seccion #76

- Añadiendo interactividad con el boton de menu mobile a traves de javascript

CarlosJCD committed:
Video #654 - Seccion #76

Añadiendo una transicion para desplegar el menu mobile

CarlosJCD committed:
Video #655 - Seccion #76

Añadiendo un boton para ocultar el menu mobile

CarlosJCD committed:
Videos #656 y #657 - Seccion #76 (FIN)

- Creando un boton para cerrar sesion
- Arreglando detalles respecto a las clases de "mostrar" y "ocultar" en distintas resoluciones

CarlosJCD committed:
Video #658 - Seccion #77 (INICIO)

Creando el formulario para la seccion de 'perfil'

CarlosJCD committed:
Videos #659 al #661 - Seccion #77

- Validacion sintactica de los campos del formulario de perfil
- Validacion logica del correo electronico
- Almacenando los nuevos datos en la base de datos
- Reflejando los nuevos datos en la interfaz

CarlosJCD committed:
Video #662 - Seccion #77

Creando el formulario para cambiar la contraseña desde el perfil

CarlosJCD committed:
Videos #663 y #664 - Seccion #77

- Validando que  password actual sea el correcto
- Validando que se haya ingresado un nuevo password
- Validando que la longitud del password nuevo sea mayor a 6 digitos

CarlosJCD committed:
Video #665 - Seccion #77 (FIN)

- Guardando la contraseña nueva en la base de datos.
- Retornando una alerta segun si se puedo guardar exitosamente la contraseña nueva o si hubo un error.

CarlosJCD committed:
Video #666 - Seccion #78 (INICIO)

Habilitando modo oscuro por medio de un media querie

CarlosJCD committed:
Video #667 - Seccion #78

Adaptando la seccion de login, crear cuenta y olvide Password a modo oscuro.

CarlosJCD committed:
Video #668 - Seccion #78

Aplicando estilos a la seccion del dashboard.

CarlosJCD committed:
Video #669 - Seccion #78

Adaptando modo oscuro a la seccion individual de proyecto.

CarlosJCD committed:
Video #670 - Seccion #78 (FIN)

Adaptando el modal de tareas a modo oscuro.

CarlosJCD committed:
Cambios: Seccion de tareas

Cambio de doble click a un click:
 - Cambiar el estado de una tarea.
 - Desplegar el modal para eliminar una tarea.
 - Desplegar el modal para editar el nombre de una tarea.

Cuando se pasa el cursor sobre los dos botonesde accion  o sobre el nombre de la tarea, este cambia a 'pointer'.

CarlosJCD committed:
Cambios: Seccion de proyectos

Ahora el redireccionamiento a la pagina del proyecto ocurre al dar click al contenedor del proyecto y no solo al texto del mismo.

Ahora al pasar el cursor sobre el contenedor del proyecto, este cambia a 'pointer'.

CheloChidoone reviewed (changes requested):
@CarlosJCD Muy buen avance pero quedo un detallito en tus modificaciones que te pidió Fernando, te digo los detalles en discord

CarlosJCD committed:
Correccion: Cambio de estado de una nueva tarea

Correccion al error que hacia que al crear una nueva tarea, la primera vez que se quiera cambiar de estado, este no cambie correctamente.
